qa-pipelines/tasks/run-test.sh

```diff
@@ -23,7 +23,9 @@ fi
 set -o nounset
 set -o allexport
 # Set this to skip a test, e.g. 011
-EXCLUDE_BRAINS_PREFIX=''
+EXCLUDE_BRAINS_PREFIX=
```

Comment:
Wasn't " " better way to handle the default case?

Reply:
It's useful to leave this here, with a blank value, because we change this when there are, for example, failing brains tests with a build, or new tests which we expect to skip. Without a value, nothing is excluded, but it does provide a stub and a clear place for us to modify this later if necessary